docs/api-types/certificate/README.md

```diff
@@ -0,0 +1,32 @@
+# Certificates
+
+cert-manager has the concept of 'Certificates' that define a desired X.509 certificate. `Certificate` resources reference an `Issuer` or `ClusterIssuer` for information on how to obtain the certificate.
+
+A simple `Certificate` could be defined as:
+
+```yaml
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: acme-crt
+spec:
+  secretName: acme-crt
```

Comment:
Can we rename the secret to something that isn't the same as the name of the Certificate? I know it makes no difference, but I want to communicate that these two things needn't be the same